### ISSUE TITLE
[Crash]: 

### ISSUE THREAD

**Issue by Phaselockd** [AllTheMods/ATM-6#3183]
### Possible Fixes

Yes

### Modpack Version

1.8.28

### What happened?

I placed an ender fluid tank and added the diamond to it and instantly crashed back to the launcher.

### Crash Log

<script src="https://gist.github.com/Phaselockd/5def10ccdcb98dba2750a77e2ec2389e.js"></script>

**Comment by Phaselockd** [comment]
It only happened on the server and not SP. We narrowed it down to be an issue with TOP Addon

**Comment by jeremiahwinsley** [comment]
Yes this is a known issue with TOP and Ender Tanks, unfortunately as far as I know there is no fix available from either side.
